packages/js/product-editor/src/components/prepublish-panel/prepublish-panel.tsx

```diff
@@ -55,6 +55,26 @@ export function PrepublishPanel( {
 			'woocommerce'
 		);
 	}
+	const panelRef = useRef< HTMLDivElement >( null );
+
+	function handleClickOutside( event: MouseEvent ) {
+		if (
+			panelRef.current &&
+			! panelRef.current.contains( event.target as Node )
+		) {
+			closePrepublishPanel();
+		}
+	}
+
+	useEffect( () => {
+		if ( ! isPublishedOrScheduled ) {
+			return;
+		}
+		document.addEventListener( 'mousedown', handleClickOutside );
```

Comment:
Why didn't you use the `mouseup` event, or the `click` event? This provides the chance to the user to 'abort' a click that was made by mistake. I think we should use it, that's what we use for most events.

Reply:
Done in commit [5be92a0](https://github.com/woocommerce/woocommerce/pull/45397/commits/5be92a0c1f79e3b0f34fbb070bc1c2fb2437da7b)